### PR TITLE
add schema inclusion

### PR DESCRIPTION
**Pull request by khaya-zulu**

### CODE REVIEW

File: docs/03-getting-started/03-data-operations.md [hunk 1]
```diff
@@ -26,12 +27,12 @@ export default () => {
...
-      }
-    }
-  }
-}
+          email: "user_" + row.id + "@example.org",
+        };
+      },
+    },
+  };
+};
```
Comment (peterp) [inline docs/03-getting-started/03-data-operations.md]: Weird? Is there something auto-formatting these for you?

Comment (khaya-zulu) [reply docs/03-getting-started/03-data-operations.md]: Yeah, it was my vscode formatting settings. Had to update it.

**Comment by peterp** [comment]
@khaya-zulu Can you fix the merge conflicts here?

**Review by jgoux — APPROVED** [review]
Do we have something for extensions exclusion already?

**Comment by khaya-zulu** [comment]
> Do we have something for extensions exclusion already?

Thank you, I cmd+F on every page, but could find reference to it

**Pull request merged**